lib/apiservers/engine/backends/eventmonitor.go

```diff
@@ -225,8 +227,8 @@ func (p DockerEventPublisher) PublishEvent(event plevents.BaseEvent) {
 		// go routine
 		go func() {
 			attrs := make(map[string]string)
-			// get the containerProxy
-			code, err := NewContainerProxy(PortLayerClient(), PortLayerServer(), PortLayerName()).exitCode(vc)
+			// get the containerEngine
+			code, err := NewContainerBackend().containerProxy.exitCode(vc)
```

Comment:
I don't like access to the containerProxy directly. I think Container should implement ExitCode or something.